doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,139 @@
+.. _gh_labels:
+
+Labeling issues and pull requests in GitHub
+###########################################
+
+The project uses GitHub issues and pull requests (PRs) to track and manage
+daily and long-term work and contributions to the Zephyr project. We use
+GitHub **labels** to classify and organize these issues and PRs by area, type,
+priority, and more, making it easier to find and report on relevant items.
+
+All GitHub issues or pull requests must be appropriately labeled.
+Issues and PRs often have multiple labels assigned,
+to help classify them in the different available categories.
+When reviewing a PR, if it has missing or incorrect labels, maintainers shall
+fix it.
+
+This saves us all time when searching, reduces the chances of the PR or issue
+being forgotten, speeds up reviewing, avoids duplicate issue reports, etc.
+
+These are the labels we currently have, grouped by type:
+
+Area:
+*****
+
+=============  ===============================================================
+Labels         ``Area:*``
+Applicable to  PRs  and issues
+Description    Indicates subsystems (e.g., Kernel, I2C, Memory Management),
+               project functions (e.g., Debugging, Documentation, Process),
+               or other categories (e.g., Coding Style, MISRA-C)  affected by
+               the bug or pull request.
+=============  ===============================================================
+
+An area maintainer should be able to filter by an area label and
+find all issues and PRs which relate to that area.
+
+Platform
+*********
+
+=============  ===============================================================
+Labels         ``Platform:*``
+Applicable to  PRs  and issues
+Description    An issue or PR which affects only a particular platform
+=============  ===============================================================
+
+To be discussed in a meeting
+****************************
+
+=============  ===============================================================
+Labels         ``API``, ``dev-review``, ``TSC``
```

Comment:
There's an organized person in me that would like to have all labels with a prefix (e.g., ``area: API``, ``meetings: API``) so that the labels are grouped together (``meetings: API``, ``meetings: Dev-review``, ``meetings: TSC``), otherwise they don't sort together.

Reply:
I agree :)

Reply:
Would it be reasonable to, by now, have this PR start as documentation of what there actually is, and then open an issue with this different fixes? (and have them applied once agreed in a a following PR). I just don't want this PR to drag long.

Reply:
OK.  Maybe you can create an issue to track this (adding a prefix to all labels).

Reply:
dont like the meeting prefix, it does not have to be meeting, this is something that can always be brought into discussion, on slack, in email and other places.

Reply:
maybe ``review:`` then?